sdk/src/RealUserMonitor.ts

```diff
@@ -122,7 +135,7 @@ export default class RealUserMonitor {
    */
   sendCustomRUMEvent(eventType: RealUserMonitoringTimings, name: string, duration: number) {
     if (eventType === RealUserMonitoringTimings.ViewLoaded) {
-      this.sendViewLoadedEvent(name, duration);
+      this.sendViewLoadedEvent({ "duration": duration, "name": name});
```

Comment:
Nit: The object literal creation can be shortened since key/values have the same name: 

```ts
this.sendViewLoadedEvent({ duration, name });
```